src/Ansatz/Chain.jl

```diff
@@ -30,52 +30,110 @@ function Chain(tn::TensorNetwork, sites, args...; kwargs...)
     Chain(Quantum(tn, sites), args...; kwargs...)
 end
 
-function Chain(::State, boundary::Periodic, arrays::Vector{<:AbstractArray})
+defaultorder(::State) = (:o, :l, :r)
+defaultorder(::Operator) = (:o, :i, :l, :r)
```

Comment:
This should be for `Chain`
```suggestion
defaultorder(::Type{Chain}, ::State) = (:o, :l, :r)
defaultorder(::Type{Chain}, ::Operator) = (:o, :i, :l, :r)
```